spacy/tests/test_gold.py

```diff
@@ -339,7 +336,7 @@ def test_gold_biluo_misaligned(en_vocab, en_tokenizer):
     gold_words = ["I", "flew to", "San", "Francisco Valley", "."]
     example = Example.from_dict(doc, {"words": gold_words, "entities": entities})
     ner_tags = example.get_aligned_ner()
-    assert ner_tags == [None, None, "B-LOC", "L-LOC", "O"]
+    assert ner_tags == ["B-ORG", "L-ORG", "B-LOC", "L-LOC", "O"]
 
```

Comment:
Also here I think the `None`'s are more appropriate as the gold entity can not be mapped back to the original words.

Reply:
Can we update the example so it's more realistic? I keep getting confused by the text

Reply:
Yea, I know, let me push a quick update.

Reply:
So basically in `predicted`, the first two tokens are "Mr. and Mrs." and "Smith", but the NER annotation reads "Mrs. Smith". I think the first token should probably be `None`, as "Mr" was never annotated. You could argue about the second token "Smith", whether it should inherit the label from "Mrs. Smith", but I can imagine contexts where half of the entity doesn't make sense to be an entity by itself.

Reply:
Thanks, I see the problem. I suggested a fix.

We could go a bit further and strip internal whitespace from the spans, to account for stuff like extra newlines. But if the underlying strings match I thing the code is correct.